Sources/TuistSupport/Constants.swift

```diff
@@ -60,17 +60,31 @@ public enum Constants {
     /// These variables are not supposed to be used by end users
     /// But only eg. for acceptance tests and other cases needed internally
     public enum EnvironmentVariables {
-        public static let verbose = "TUIST_VERBOSE"
-        public static let colouredOutput = "TUIST_COLOURED_OUTPUT"
-        public static let versionsDirectory = "TUIST_VERSIONS_DIRECTORY"
+        public static let verbose = "TUIST_CONFIG_VERBOSE"
+        public static let colouredOutput = "TUIST_CONFIG_COLOURED_OUTPUT"
+        public static let versionsDirectory = "TUIST_CONFIG_VERSIONS_DIRECTORY"
         @available(*, deprecated, message: "This is used only for acceptance tests, see https://github.com/tuist/tuist/issues/2777")
-        public static let forceConfigCacheDirectory = "TUIST_FORCE_CONFIG_CACHE_DIRECTORY"
-        public static let automationPath = "TUIST_AUTOMATION_PATH"
-        public static let queueDirectory = "TUIST_QUEUE_DIRECTORY"
-        public static let cloudToken = "TUIST_CLOUD_TOKEN"
-        public static let cacheManifests = "TUIST_CACHE_MANIFESTS"
-        public static let statsOptOut = "TUIST_STATS_OPT_OUT"
-        public static let githubAPIToken = "TUIST_GITHUB_API_TOKEN"
+        public static let forceConfigCacheDirectory = "TUIST_CONFIG_FORCE_CONFIG_CACHE_DIRECTORY"
+        public static let automationPath = "TUIST_CONFIG_AUTOMATION_PATH"
+        public static let queueDirectory = "TUIST_CONFIG_QUEUE_DIRECTORY"
+        public static let cloudToken = "TUIST_CONFIG_CLOUD_TOKEN"
+        public static let cacheManifests = "TUIST_CONFIG_CACHE_MANIFESTS"
+        public static let statsOptOut = "TUIST_CONFIG_STATS_OPT_OUT"
+        public static let githubAPIToken = "TUIST_CONFIG_GITHUB_API_TOKEN"
+        public static let detailedLog = "TUIST_CONFIG_DETAILED_LOG"
+        public static let osLog = "TUIST_CONFIG_OS_LOG"
+
+        public static let deprecatedVerbose = "TUIST_VERBOSE"
```

Comment:
Is it ok if we remove those in 2.0?

Reply:
yes, once we merge these I'll open a PR there